objects.js

```diff
@@ -8496,17 +8497,19 @@ WatcherMorph.prototype.userMenu = function () {
         );
         if (this.currentValue &&
                 (isString(this.currentValue) || !isNaN(+this.currentValue))) {
-            menu.addItem(
-                'export...',
-                function () {
-                    var ide = myself.parentThatIsA(IDE_Morph);
-                    ide.saveFileAs(
-                        myself.currentValue.toString(),
-                        'text/plain;charset=utf-8',
-                        myself.getter // variable name
-                    );
-                }
-            );
+            menu.addItem('export...', this.valueExporter('plain'));
+        } else if (this.currentValue instanceof List) {
+            subMenu = new MenuMorph(this.currentValue);
+            if (!this.currentValue.contents.some(
```

Comment:
Why don't we make a function `isOneDimension` on `List`?

Also if a list is a single dimension, then it seems like CSV and Plain text should be the same output - I'd merely like the menu text to be more clear.